app/models/user_stats.rb

```diff
@@ -183,7 +183,9 @@ def get_applicable_field(obj)
     # until all records have a value for each column except id and user_id.
     # At the end, we find the existing user_stats record id by user_id, and
     # add in the id and user_id, and update all records at once.
-    def refresh_all_user_stats
+    def refresh_all_user_stats(dry_run: false)
+      return if dry_run
+
```

Comment:
@JoeCohen I'm going to alter so that it returns messages like the other "refreshers"